### DISCUSSION TITLE
catch up tv

### DISCUSSION THREAD

**Discussion by DoubleJ19** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1087] [Q&A] [answered]
bonjour,

J'ai un problème sur catch up tv avec le groupe m6 play  avec le message suivant: "Echec de lecture d’un ou plusieurs fichiers. Consulter le journal pour plus d’informations sur ce message"
Avec aussi BFMTV dont le message est le suivant: "Accès à ce contenu nécessite un compte RMCBFMPlay . Vous pouvez configurer votre ID et MDP dans la configuration de cette extension. Si vous n'avez pas de compte ,vous pouvez le créer à cette URL 
"https://rmcbfmplay.com"
Et pour finir France 3 où je n'ai pas de son

Pourriez-vous me dire quelles sont les démarches à suivre pour résoudre mes problèmes.
En vous remerciant
DoubleJ19

**Comment by joaopa00** [comment, marked as answer]
Il faut reporter le bug dans la partie Issue.
